script/conflate_osm.sh

```diff
@@ -10,7 +10,7 @@ DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd );
 # ---- pbf2json ----
 
 # location of pbf2json binary
-PBF2JSON_BIN=${PBF2JSON_BIN:-"pbf2json.linux-x64"};
+PBF2JSON_BIN=${PBF2JSON_BIN:-"$DIR/../node_modules/pbf2json/build/pbf2json.linux-x64"};
```

Comment:
hmm.. this is tied to a specific OS, it would be nicer if we could detect the host OS and choose the correct binary.

possibly out-of-scope for this PR, maybe just open an issue to fix it later?